templates/web/base/waste/garden/subscribe_details.html

```diff
@@ -18,7 +18,16 @@
         data-per_bin_cost="[% per_bin_cost %]"
         data-per_new_bin_first_cost="[% per_new_bin_first_cost %]"
         data-per_new_bin_cost="[% per_new_bin_cost %]"
+        [% IF is_staff && waste_features.ggw_discount_as_percent %]
+          data-discount_applied="[% form.saved_data.apply_discount %]"
```

Comment:
I may be missing something here - but if you didn't have the data-discount_applied line here (and sacks equivalent)  for the new subscription page, you wouldn't need the special precalculated_discount flag in the JS? As that's only used when this line is present (and this line isn't used for anything else).